### ISSUE TITLE
IE does not support switchToFrame

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [dequelabs/axe-webdriverjs#71]
🙃

Running this with `jest-environment-webdriver` and error reporting in https://github.com/SeleniumHQ/selenium/pull/6321:

```
 FAIL  test/end-to-end/dev/Tests/Consumption/ConsumptionAccessibility.test.ts (35.275s)
  Consumption Accessibility
    × passes aXe (7426ms)

  ● Sample Accessibility Tes › passes aXe

    UnsupportedOperationError: switchToFrame: Error 404: Not Found
    Not Found

      at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:557:11)
      at Executor.execute (node_modules/selenium-webdriver/lib/http.js:468:26)
```

Looks like it's from the `driver.switchTo()s` in `findFramesAndInject` in `lib/inject.js`. Is there a way around using them? Can the script be injected using, say, `driver.executeScript` and some fancy string concats?

**Comment by stephenmathieson** [comment]
Is there any way you could provide a more complete set of reproduction steps? The [`switchToFrame` method](https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/remote/server/handler/SwitchToFrame.html) is part of Selenium core, so I'm struggling to understand why it'd 404.

**Comment by JoshuaKGoldberg** [comment]
> part of Selenium core

I assume it's because the IE driver doesn't support that method. 

You should be able to repro using the sample code listed in the README.md [Usage](https://github.com/dequelabs/axe-webdriverjs#usage) section when running against Internet Explorer using IEDriverServer.exe.

If it helps, I'm running on Windows Server 2016 against IE 11.

**Comment by stephenmathieson** [comment]
Thanks, this helps. I'll see about getting a Windows machine for testing and/or use a VM.

**Comment by stephenmathieson** [comment]
I've done some primitive testing and research, and am not able to reproduce this. What version of IE driver are you using?

**Comment by JoshuaKGoldberg** [comment]
I'm on https://selenium-release.storage.googleapis.com/3.9: IEDriverServer_Win32_3.9.0.zip.

**Comment by stephenmathieson** [comment]
I still haven't been able to reproduce this. Is there any way you can setup a minimal reproduction repo/gist?

**Comment by JoshuaKGoldberg** [comment]
I haven't been able to 😕 so I guess it's probably something convoluted on my end... ah well. Thanks for testing!
